inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/listener/RestartStreamListener.java

```diff
@@ -89,9 +89,11 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
 
         Map<String, String> kvConf = new HashMap<>();
         groupExtList.forEach(groupExtInfo -> kvConf.put(groupExtInfo.getKeyName(), groupExtInfo.getKeyValue()));
-        streamExtList.forEach(extInfo -> {
-            kvConf.put(extInfo.getKeyName(), extInfo.getKeyValue());
-        });
+        if (streamExtList != null) {
```

Comment:
// import org.apache.commons.collections.CollectionUtils;
```suggestion
        if (!CollectionUtils.isEmpty(streamExtList)) {
```